build.py

```diff
@@ -0,0 +1,74 @@
+# A nasty basic implementation with no tests to preserve the ability to build data required by the
+# service.
+
+import glob
+import json
+import pathlib
+import sys
+
+
+OUTPUT = "./build"
```

Comment:
I know the script is in "WIP" state, but maybe we should make this relative to the script path, to avoid accidentally overriding other directory if invoking from a different path.

Reply:
The original `build.sh` script worked the same way. Moreover, no one is expected to use this version of the script, really. It is meant to demonstrate how the data for the service is built from the source data; essentially a substitute for better CONTRIBUTING.md. I would prefer not to delay this MR with a discussion of what the CLI should look like so that we don't block Jirka's work.

Reply:
Resolved in person. `build.py` will keep working in the current working directory by default for now.